cmd/cli/cmd/alpha/manifest-gen/handlers.go

```diff
@@ -0,0 +1,92 @@
+package manifestgen
+
+import (
+	"strings"
+
+	"capact.io/capact/cmd/cli/cmd/alpha/manifest-gen/attributes"
+	"capact.io/capact/cmd/cli/cmd/alpha/manifest-gen/common"
+	"capact.io/capact/cmd/cli/cmd/alpha/manifest-gen/implementations"
+	"capact.io/capact/cmd/cli/cmd/alpha/manifest-gen/interfaces"
+	"capact.io/capact/internal/cli/alpha/manifestgen"
+	"capact.io/capact/pkg/sdk/apis/0.0.1/types"
+	"github.com/pkg/errors"
+	"k8s.io/utils/strings/slices"
+)
+
+type genManifestFn func(opts common.ManifestGenOptions) (manifestgen.ManifestCollection, error)
+
+func generateAttribute(opts common.ManifestGenOptions) (manifestgen.ManifestCollection, error) {
+	files, err := attributes.GenerateAttributeFile(opts)
+	if err != nil {
+		return nil, errors.Wrap(err, "while generating attribute file")
+	}
+	return files, nil
+}
+
+func generateType(opts common.ManifestGenOptions) (manifestgen.ManifestCollection, error) {
+	if slices.Contains(opts.ManifestsType, string(types.InterfaceManifestKind)) {
+		inputTypeManifest, err := interfaces.GenerateInterfaceFile(opts, manifestgen.GenerateInputTypeTemplatingConfig)
+		if err != nil {
+			return nil, errors.Wrap(err, "while generating input type templating config")
+		}
+		if !slices.Contains(opts.ManifestsType, string(types.ImplementationManifestKind)) {
```

Comment:
Remove this if: If Type selected along with Interface + Implementation => also generate Interface output Type